src/main/java/io/kestra/plugin/airtable/records/Create.java

```diff
@@ -0,0 +1,249 @@
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a new record in an Airtable table",
+    description = "Create one or more new records in an Airtable table. Can create a single record or multiple records (max 10) in one operation."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Create a single task record",
+            full = true,
+            code = """
+                id: create_airtable_task
+                namespace: company.airtable
+
+                tasks:
+                  - id: create_task
+                    type: io.kestra.plugin.airtable.records.Create
+                    baseId: "{{ secret('AIRTABLE_BASE_ID') }}"
+                    tableId: "Tasks"
+                    apiKey: "{{ secret('AIRTABLE_API_KEY') }}"
+                    fields:
+                      "Task Name": "Implement new feature"
+                      "Status": "Todo"
+                      "Priority": "High"
+                      "Due Date": "2024-12-31"
+                    typecast: true
+                """
+        ),
+        @Example(
+            title = "Create multiple customer records",
+            full = true,
+            code = """
+                id: create_customers
+                namespace: company.airtable
+
+                tasks:
+                  - id: create_customer_records
+                    type: io.kestra.plugin.airtable.records.Create
+                    baseId: "{{ secret('AIRTABLE_BASE_ID') }}"
+                    tableId: "Customers"
+                    apiKey: "{{ secret('AIRTABLE_API_KEY') }}"
+                    records:
+                      - "Name": "Acme Corp"
+                        "Email": "contact@acme.com"
+                        "Status": "Active"
+                      - "Name": "Beta Inc"
+                        "Email": "hello@beta.com"
+                        "Status": "Prospect"
+                    typecast: true
+                """
+        ),
+        @Example(
+            title = "Create record from dynamic data",
+            full = true,
+            code = """
+                id: create_from_input
+                namespace: company.airtable
+
+                tasks:
+                  - id: create_dynamic_record
+                    type: io.kestra.plugin.airtable.records.Create
+                    baseId: "{{ secret('AIRTABLE_BASE_ID') }}"
+                    tableId: "Orders"
+                    apiKey: "{{ secret('AIRTABLE_API_KEY') }}"
+                    fields:
+                      "Customer": "{{ inputs.customer_name }}"
+                      "Product": "{{ inputs.product }}"
+                      "Amount": "{{ inputs.amount }}"
```

Comment:
You need to add an `inputs:` block before your `tasks:` block in this example then